$NetBSD: patch-ak,v 1.1 2007/03/17 15:59:12 rillig Exp $

g++4 complains:
src/PixmapTimer.h:87: error: extra qualification 'CPixmapTimer::' on member 'CPixmapTimer'

--- src/PixmapTimer.h.orig	2005-07-25 22:13:55.000000000 +0200
+++ src/PixmapTimer.h	2007-03-17 16:02:46.000000000 +0100
@@ -84,7 +84,7 @@ class CPixmapTimer : public QObject
 
 public:
 	CPixmapTimer(QObject * parent = 0, const char * name = 0);
-	CPixmapTimer::~CPixmapTimer();
+	~CPixmapTimer();
 
 signals:
 	void signalTimeout(const QPixmap &);
